plugins/Rockets/RocketsPlugin.cpp

```diff
@@ -294,16 +286,20 @@ class RocketsPlugin::Impl : public ActionInterface
                          const std::function<PropertyMap(PropertyMap)>& action)
     {
         _bindEndpoint(desc.methodName, [
-            name = desc.methodName, input, action, &engine = _engine
+            name = desc.methodName, action, &engine = _engine
         ](const auto& request) {
-            PropertyMap params = input;
-            if (::from_json(params, request.message))
+            engine.triggerRender();
+            try
             {
-                engine.triggerRender();
-                return Response{to_json(action(params))};
+                return Response{
+                    to_json(action(jsonToPropertyMap(request.message)))};
```

Comment:
Is the code truly equivalent? I mean, before action was invoked with default values coming from input, but now those values are gone. Doesn't this have any unexpected side effect?

Reply:
It was actually wrong before :)